terraform/concourse/outputs.tf

```diff
@@ -9,3 +9,11 @@ output "concourse_security_group" {
 output "concourse_security_group_id" {
   value = "${aws_security_group.concourse.id}"
 }
+
+output "concourse_elb_name" {
+  value = "${aws_elb.concourse.name}"
+}
+
+output "concourse_dns_name" {
+  value = "${aws_route53_record.concourse.name}"
```

Comment:
Is this output actually used anywhere?  If not, can we remove it until it is needed.